torch/onnx/symbolic_opset13.py

```diff
@@ -462,6 +463,50 @@ def reduce_dim(g, self, dim, keepdim, dtype):
     return reduce
 
 
+@_onnx_symbolic("aten::unflatten")
+@_beartype.beartype
+def unflatten(g: jit_utils.GraphContext, input, dim, unflattened_size):
```

Comment:
Comment why this requires minimum opset 13?

Reply:
I tried to put this in Opset9, and turns out ops like Concat, Slice, .., has moved attrs to inputs, and that complicates things in a way that we have to engage helper functions. By demand, opset 13 should be ebough for current requests. I will put a comment here.